### ISSUE TITLE
Learning Rate Scheduler is fixed to CosineLRScheduler

### ISSUE THREAD

**Issue by HCookie** [ecmwf/anemoi-core#510]
### Is your feature request related to a problem? Please describe.

Learning Rate Scheduler is fixed to CosineLRScheduler, this should be configurable for experimentation with the default being CosineLRScheduler.

### Organisation

ECMWF

**Comment by anaprietonem** [comment]
Related to this it could be also be nice to check if we can use PyTorch's CosineLR scheduler https://docs.pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingLR.html rather than the one from timm. That way we could remove anemoi's dependency on that library

**Comment by VeraChristina** [comment]
Do we consider this closed?
Optimizer configurability: https://github.com/ecmwf/anemoi-core/pull/588
LR scheduler configurability: https://github.com/ecmwf/anemoi-core/pull/986

**Comment by anaprietonem** [comment]
Good catch Vera, I agree this is now completed. @HCookie  if you disagree feel free to reopen.
